sos_trades_core/sos_wrapping/test_discs/disc_distortion.py

```diff
@@ -0,0 +1,69 @@
+'''
+Copyright 2022 OS-Climate https://os-climate.org
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+from sos_trades_core.execution_engine.sos_discipline import SoSDiscipline
+from sympy.series import limitseq
+
+class DiscDistortion(SoSDiscipline):
+    '''
+    Testing discipline
+    Implements guitar pedal effect "distortion" (clipping)
+    '''
+
+    # ontology information
+    _ontology_data = {
+        'label': 'sos_trades_core.sos_wrapping.test_discs.disc_distortion',
+        'type': 'Fake',
+        'source': 'SoSTrades Project',
+        'validated': '',
+        'validated_by': '',
+        'last_modification_date': '',
+        'category': '',
+        'definition': '',
+        'icon': '',
+        'version': '',
+    }
+
+    _maturity = 'Fake'
```

Comment:
how is `_maturity` used?

Reply:
Maturity level is used to be displayed in the GUI to understand the maturity of a model: Is it and official one, a research one or test/fake one?

Reply:
👍